docs/UserGuide.md

```diff
@@ -50,110 +70,181 @@ Some example commands you can try:
+
+* A whitespace must be included before every prefix.<br>
+  e.g. `n/Knives Out t/Suspense` is acceptable but `n/Knives Outt/Suspense` is not.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/<NAME> [t/<TAG>]`, `[t/<TAG>] n/<NAME>` is also acceptable.
+  e.g. if the command specifies `n/NAME [t/TAG]`, `[t/TAG] n/NAME` is also acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `n/You n/Me`, only `n/Me` will be taken.
+* All **names must be unique** and duplicates will be ignored.<br>
+  e.g. if you try to add `n/Inception` into the show list that already contains that show, there will be a message telling you that this show already exists in the list.
+
+* If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+  e.g. if you specify `n/Batman n/Superman`, only `n/Superman` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `exit` and `list` ) will be ignored.<br>
   e.g. if the command specifies `exit 123`, it will be interpreted as `exit`.
 
+* The **index** parameter provided should be a [non-zero unsigned integer](https://en.wikipedia.org/wiki/Integer_(computer_science)) within the allowed range of Java’s [`int`](#glossary) data type. On top of that, the index should be within the bounds of the show list.<br>
+  e.g. If there are 5 shows saved in the show list, then the valid index ranges from 1 to 5. 
+
 </div>
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
+## Features
+
 ### Adding a show: `add`
 
-Adds a new show to Trackermon. Note that the name of the show can only contain alphanumeric characters.
+**Description:** Planning to watch a show and want to store its details? Add a show to Trackermon's show list!
 
-Format: `add n/<NAME> s/<STATUS> [t/<TAG>]…​`
+**Format:** `add n/NAME s/STATUS [c/COMMENT] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-warning">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A show can have any number of comments or tags (including 0)
 </div>
 
-Examples:
-* `add n/All of us are dead s/completed`
-* `add n/All of us are dead s/completed t/Kdrama`
+**Example & Output:** `add n/All of us are dead s/plan-to-watch t/Horror`
+
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
 ### Listing all shows: `list`
 
-Shows a list of all shows in Trackermon.
+**Description:** Wanting to view all your shows at once? Display a list of shows in Trackermon's show list!
+
+**Format:** `list`
 
-Format: `list`
+**Example & Output:** `list`
+
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
```

Comment:
I feel that listing doesnt need a image.

Reply:
Noted.